src/Compute/Compute/Manual/EncryptionSettings.cs

```diff
@@ -0,0 +1,41 @@
+//
+// Copyright (c) Microsoft and contributors.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+// Warning: This code was generated by a tool.
+//
+// Changes to this file may cause incorrect behavior and will be lost if the
+// code is regenerated.
+
+namespace Microsoft.Azure.Management.Compute.Models
+{
+    public partial class EncryptionSettings
+    {
+        public EncryptionSettings(EncryptionSettingsCollection encryptionSettingsCollection)
+        {
+            this.Enabled = (encryptionSettingsCollection == null) ? (bool?) null : encryptionSettingsCollection.Enabled;
```

Comment:
@hyonholee if we are adding the null-check, it should be around the entire constructor since `encrypstionSettingsCollection` is used in multiple places here